src/inject/dynamic-theme/style-manager.ts

```diff
@@ -422,6 +428,11 @@ export function manageStyle(element: StyleElement, {update, loadingStart, loadin
         pause();
         removeNode(corsCopy);
         removeNode(syncStyle);
+        loadingEnd();
+        if (loadingLinkID && loadingLinkStyles.has(loadingLinkID)) {
```

Comment:
I think you need a boolean here, e.g. `isLinkLoading` instead of `loadingLinkID`. Or maybe don't need at all.